server/routes/chore.js

```diff
@@ -1,20 +1,23 @@
 import express from "express";
 import { body } from "express-validator";
+// CONTROLLER FUNCTIONS
+import {
+  createChore,
+  getAllChores,
+  getSingleChore,
+} from "../controller/chore.js";
 // MIDDLEWARE
 import { verifyToken } from "../middleware/auth.js";
-// CONTROLLER FUNCTIONS
-import { createChore, getAllChores } from "../controller/chore.js";
 
 const router = express.Router();
 
 //ROUTES
-// router.get("/id", ); //get a single chore
+router.get("/id/:id", verifyToken, getSingleChore); //get a single chore
```

Comment:
Does this need to be parameterized to `/:id`?

Reply:
It is, `/id/:id`

Reply:
Oh whoops, I mean like don't have the first ID there. Just have the path be `/api/chores/:id`.

Reply:
I'll make the change